src/tools/vdorecover/vdorecover

```diff
@@ -242,7 +242,8 @@ _repointUpperDevicesOrMountVDO(){
 		dmsetup resume $VDO_DEPENDENT_DEV
 		echo "You may want to remount, and run fstrim, on any filesystem"
 	        echo "mounted on ${VDO_DEPENDENT_DEV}."
-		_fstrimAndPrompt "the device or filesystem atop ${VDO_DEPENDENT_DEV}"
+                DEPENDENT_MOUNT=$(awk "/${VDO_DEPENDENT_DEV}/ {print \$2}" /proc/self/mounts)
```

Comment:
Will the trailing '/' always be there? The listings I see in /proc/self/mounts don't have a trailing '/'.

Reply:
Are you talking about the `/${VDO_DEPENDENT_DEV}/`?  That's a regex.

Reply:
Yeah - 0kay. When I tried it manually in a terminal for something listed in /proc/self/mounts it gave me a syntax error.